pkg/client/images.go

```diff
@@ -22,22 +22,22 @@ import (
 // ImagesClient defines the image client interface
 type ImagesClient interface {
 	// Images
-	CreateImagectx(ctx context.Context, image *v1.Image) (*v1.Image, error)
```

Comment:
This seems like this was a bug. Did we really never try to use the ImagesClient interface? The 'implementing' struct seems to have always used `CreateImage` instead of `CreateImagesctx`

Reply:
Yeah that's a bug, we haven't really used this interface yet, as function manager requires only ’GetImage’, and declares it's own interface for that. This interface would eventually be used in CLI

Reply:
Yeah, we only use the GetImage method in Function Manager.